lib/locales/fr.ts

```diff
@@ -244,7 +251,7 @@ export const locale: Locale = {
     casimir_of_poland: 'Saint Casimir († 1484)',
     catherine_of_alexandria_virgin: 'Sainte Catherine d’Alexandrie, vierge et martyre († IVème s.)',
     catherine_of_saint_augustine_de_simon_de_longpre_virgin:
-      'Bienheureuse Catherine de Saint-Augustin, religieuse hospitalière de la Miséricorde († 1668)',
+      'Bienheureuse Catherine de Saint-Augustin, vierge († 1668)', // src: mr_fr_2021_ed3, mr_fr_1982_ed2_coutances
```

Comment:
I have correctly mentioned the two sources here `mr_fr_2021_ed3`, `mr_fr_1982_ed2_coutances`, since this is the official translation found both in the French-language Roman Missal (which applies to the French-speaking part of Canada) and in the Proper of the Diocese of Coutances.